recipe/meta.yaml

```diff
@@ -48,40 +48,40 @@ requirements:
     # above, we need most of these at build time, so they are mostly
     # host-like
     - boost =1.70
-    - cfitsio
-    - eigen
+    - cfitsio =3
+    - eigen =3.3
```

Comment:
Just a note - we really want eigen 3.4 but who knows when it will come out or what it will break.